watchtower/wtdb/codec_test.go

```diff
@@ -57,6 +178,56 @@ func TestCodec(t *testing.T) {
 		return true
 	}
 
+	customTypeGen := map[string]func([]reflect.Value, *rand.Rand){
+		"ClientSession": func(v []reflect.Value, r *rand.Rand) {
+			pol := wtpolicy.Policy{
+				BlobType:     blob.Type(r.Int31()),
+				MaxUpdates:   uint16(r.Int31()),
+				RewardBase:   uint32(r.Int63()),
+				RewardRate:   uint32(r.Int63()),
+				SweepFeeRate: lnwallet.SatPerKWeight(r.Int63()),
+			}
+
+			obj := wtdb.ClientSession{
+				SeqNum:           uint16(r.Int31()),
+				TowerLastApplied: uint16(r.Int31()),
+				TowerID:          wtdb.TowerID(r.Int63()),
+				KeyIndex:         uint32(r.Int63()),
+				Policy:           pol,
+			}
+
+			obj.RewardPkScript = make([]byte, uint8(r.Int31()))
```

Comment:
the issue is with the private key, since reflect can't set the private values of the curve. either way, this portion is now removed, since we embed a `ClientSessionBody` struct instead that doesn't have the privkey and test that on it's own